src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/integration/ReferralControllerIntegrationTest.kt

```diff
@@ -218,7 +216,6 @@ class ReferralControllerIntegrationTest : IntegrationTestBase() {
   }
 
   @Test
-  @Disabled("This test is disabled as we have removed saving of pom ")
   fun `Submitting a referral and fetching it returns staff details as part of the referral`() {
```

Comment:
We need an equivalent test the verifies that a referral submission still functions as expected in the event of the allocation manager API returning a 404 for a pom request. I have an example up and running.